src/main/java/io/confluent/connect/jdbc/source/BulkTableQuerier.java

```diff
@@ -36,8 +36,8 @@
 public class BulkTableQuerier extends TableQuerier {
   private static final Logger log = LoggerFactory.getLogger(BulkTableQuerier.class);
 
-  public BulkTableQuerier(QueryMode mode, String name, String topicPrefix) {
-    super(mode, name, topicPrefix);
+  public BulkTableQuerier(QueryMode mode, String name, JdbcSourceTaskConfig config) {
```

Comment:
4 constructor parameters is too much for my liking. It's not that it's a big deal, but unlike others it's the original value that we use.
As for other parameters - they are not vanilla values, they are derived.